Makefile.nt

```diff
@@ -18,8 +18,62 @@ include Makefile.shared
 defaultentry:
 	@echo "Please refer to the installation instructions in file README.win32.adoc."
 
+REQUIRE_FLEXDLL:=$(if $(shell which flexlink 2>/dev/null),,$(FLEXDLL_SRC)/flexlink$(EXE))
+BOOT_FLEXLINK_CMD=$(if $(REQUIRE_FLEXDLL),FLEXLINK_CMD="../boot/ocamlrun ../$(FLEXDLL_SRC)/flexlink.exe")
+# Sys.command doesn't accept / in PATHs (because cmd doesn't)
```

Comment:
It always was - just that no one would have ever configured MKEXE to include a path so there's no reason anyone would have noticed. `Sys.command` doesn't cope with forward-slashes as `cmd /c` interprets them as command line switches (that may be a bug in `Sys.command`, of course)